tests/lean/semicolonOrLinebreak.lean.expected.out

```diff
@@ -0,0 +1,5 @@
+building '/nix/store/kzswn86bwsdhxg5jb5d8vnfjkq36akl7-lean.drv'...
+semicolonOrLinebreak.lean:2:12: error: expected ';' or line break
+semicolonOrLinebreak.lean:9:31-10:8: error: field 'y' is missing
+semicolonOrLinebreak.lean:10:8: error: expected command
```

Comment:
This error message was better in Leo's proposal, but again I don't think it matters in practice. The important thing is that there is an error at all.